src/FakeItEasy.Analyzer/VisualBasic/Resources.resx

```diff
@@ -103,4 +103,10 @@
     <data name="ArgumentConstraintNullabilityMismatchMessageFormat" xml:space="preserve">
     <value>Parameter '{0}' of method '{1}' is nullable, but argument constraint '{2}' is not. Calls where this argument is Nothing will not be matched.</value>
   </data>
+  <data name="MakeConstraintNullableCodeFixTitle">
+    <value>Change constraint type to nullable</value>
+  </data>
+  <data name="MakeNotNullConstraintCodeFixTitle">
+    <value>Change to 'not Nothing' constraint</value>

```

Comment:
I don't know... We could add a `IsNotNothing` constraint, just for VB developers ;). But it would also appear in intellisense in C# code...

What about this?

> Change to 'not Nothing' constraint (IsNotNull)